elastic/kibana
# [Fleet] fix dynamic mapping missing `time_series_metric`

## Summary

Closes https://github.com/elastic/kibana/issues/155004

Found a simplification of adding `time_series_metric` to `dynamic_templates`.

I think tsdb enabled check is not needed similarly to https://github.com/elastic/kibana/pull/157047

To test:
- Install package by upload or local registry [istio-0.3.3.zip](https://github.com/elastic/kibana/files/11849494/istio-0.3.3.zip)
- Check that the created `metrics-istio.istiod_metric`s index template includes `time_series_metric`
```
"dynamic_templates": [
        {
          "istio.istiod.metrics.*.counter": {
            "path_match": "istio.istiod.metrics.*.counter",
            "mapping": {
              "time_series_metric": "counter",
              "type": "double"
            }
          }
        },
        {
```
